clang/test/CXX/drs/cwg2398.cpp

```diff
@@ -0,0 +1,115 @@
+// RUN: %clang_cc1 %s -fsyntax-only -std=c++23                                     -verify=expected,new
```

Comment:
Why only C++23 is tested? DR tests should test all language modes.

Reply:
While it's true this is a DR, I just don't think for this particular case it's worth the cost of testing every single mode, as there are no particularly interesting interactions there.
Moot point as I will be moving it away anyway.